Unity.gitignore

```diff
@@ -23,9 +23,15 @@
 # Autogenerated Jetbrains Rider plugin
 /[Aa]ssets/Plugins/Editor/JetBrains*
 
+# Plastic SCM meta and ache directory
+.plastic/
+
 # Visual Studio cache directory
 .vs/
 
+#JetBrains Rider cache directory
+.idea/
```

Comment:
@whisper3zzz per the contribution guidelines, we do not accept editor-specific paths to ignore since they are included in the [global gitignore directory](https://github.com/github/gitignore/blob/main/Global/README.md). However, we are open to having them commented out with an explanation like you see [here](https://github.com/github/gitignore/blob/main/Python.gitignore#L157-L162) 👍